cmd/fastly-exporter/main.go

```diff
@@ -106,6 +102,16 @@ func main() {
 		}
 	}
 
+	switch deprecatedSubsystem {
+	case "rt":
+		// good
+	case "origin":
+		level.Error(logger).Log("err", "-subsystem cannot be 'origin'")
```

Comment:
In this case, it might be useful to tell the user what flag they should be using for Origin Inspector?

Reply:
@hermanschaaf I'm actually not sure if we need this. 

Maybe I'm misunderstanding the original intent but it seems like `-subsystem` was meant to be deprecated entirely. In that case couldn't we just have anything but `-subsystem rt` error through the default clause below?

Reply:
Yeah, I'm also not sure about the original intent but what you're suggesting sounds good to me too.

Reply:
@hermanschaaf looking at it again I think I get it -- the "origin" case calls `os.Exit(1)` but the default just logs a warning. So it's being explicit that "origin" isn't allowed. I'll keep it the way it is 👍

Reply:
`-subsystem` was originally user-specifiable, but with the addition of Origin Inspector, the intent is that the subsystem dimension of metric names has been "claimed" by the exporter. That is, all metrics captured and exported prior to this Origin Inspector PR are in the `rt` subsystem, and all Origin Inspector metrics are in the `origin` subsystem.